governance.md

```diff
@@ -0,0 +1,101 @@
+
+- Organize and lead regular Steering Committee meetings, on a cadence to be agreed by the Committee. 
+- Work with the Committee to develop an annual operating plan and budget and report regularly on execution to plan.
+- Authorize payments to third-party vendors in accordance with the Committee-approved budget or for expenses approved by majority vote by the Committee.
+- Lead Sponsor recruitment, with the assistance of other Committee and Technical Community members.
+- Regularly attend Technical Community meetings and work closely with the Technical Community Leader to ensure ongoing alignment between the Steering Committee and Technical Community.
+
+The Treasurer will be elected according to the same criteria and process as the Director. The Treasurer will have the following responsibilities.
+
+- Advise on and validate operating plan’s budget.
+- Review payments authorized by the Director each month, and report regularly on revenues and expenditures in relation to budget.
+
+The Marketing Lead will be elected according to the same criteria and process as the Director. The Marketing Lead will have the following responsibilities.
+
+- Develop the Marketing portion, including metrics, of the annual operating plan in consultation with the Technical Community. Report regularly to the Steering Committee on progress against plan.
+- Work with the Technical Community and other contributors to develop and execute launch plans for major releases and any major events.
+- Work with the Community Manager and other contributors as needed to support ongoing marketing activities.
+
+The Technical Community Leader and Curriculum Lead’s roles and selection process are described in the Technical Community section above.  
+
+The Committee may choose to define and formalize additional roles to assist the Director or lead particular initiatives under the Voting procedures described below. 
+
+#### Voting Procedures
+
+- 1 vote per Steering Committee Voter
+- Other voting members may be admitted by majority vote of existing Voters.
+- If a Steering Committee Voter misses 3 meetings in a row without designating a proxy, their voting capacity will be suspended until they have attended 2 meetings in succession.
+- If a Steering Committee Voter misses 4 or more meetings in a row, with or without a proxy, the Steering Committee may remove that individual and replace them with someone with greater availability.
+
+Elections and any other voting decisions require a quorum of 50% of voting members. Voting can be held during Committee meetings, or by electronic means within a specified time period.
+
+Amendments to this or Policy documents require a majority vote of the Steering Committee.
+
+### Compliance with Policies and Licenses
+
+Contributors will comply with the policies adopted by the Steering Committee of NRE Labs. These policies may include:
+
+- Contributor License Agreement
```

Comment:
Is there a proposed template for that one ? Does that mean that the Project should hold the copyright for all contributions, then ?

Reply:
We've been advised to have contributors use the DCO https://developercertificate.org/ vs the heavier CLA. This will be included in the contributor documentation.